task_11/app/index.html

```diff
@@ -0,0 +1,365 @@
+              <th>Delivery</th>
+            </tr>
+            <tr>
+              <td>2 documents</td>
+              <td>2 languages</td>
+              <td value='USD:696.00'>$ 696.00</td>
+              <td>48 hours*</td>
+            </tr>
+          </table>
+          <div class="actions-right">
+              <button class="btn-action button" type="submit">Place order</a>
+              <button class="button">Print a quote</a>
+          </div>
+        </section>
+        <section class="address right">
+            <h3 class="section-title">
+                <i class="fa fa-life-ring icon-offset" aria-hidden="true"></i>
+                Help center
+              </h3>
+            <address>
+                <p>If you need assistance while placing your order, contact one of our beloved Project Managers by phone or email.</p>
+                <a href="tel:+971(0)555 555">+971(0)555 555</a>
+                <a href="mailto:help@qtest.com">help@qtest.com</a>
+            </address>
+        </section>
+      </section>
+      <div class="form-container">
+          <form id="main-form" action="send()">
+
+            <section class="selectable">
+              <h1 class="main-title">Select what you want to translate</h1>
+              <fieldset>
+                <div class="flex-container">
+                    <div class="select-type">
+                        <input required type="radio" name="select-type" id="type1" value="website">
+                        <label for="type1" class="select-wrapper">
+                          <img src="img/website.png" alt="">
+                          <span>Website</span>
+                        </label>
+                      </div>
+                      <div class="select-type">
+                          <input required  type="radio" name="select-type" id="type2" value="website"/>
+                          <label for="type2" class="select-wrapper">
+                            <img src="img/docs.png" alt="">
+                            <span>Documents</span>
+                          </label>
+                        </div>
+                        <div class="select-type">
+                            <input  required  type="radio" name="select-type" id="type3" value="website"/>
+                            <label for="type3" class="select-wrapper">
+                              <img src="img/account.png" alt="">
+                              <span>Social account</span>
+                            </label>
+                          </div>
+                  </div>
+                
+              </fieldset>
+            </section>
+
+              <div class="counter-container">
+                <section class="uploadable">
+                  <fieldset>
+                    <div class="flex-container">
+                      <h2 class="section-title">Upload your documents
+                        <span class="section-title-filetypes">(Doc, Docx, Text, PDF, Markdown)</span>
```

Comment:
I've made the same mistake in my task, but now I think `<small>` here would be slightly more semantic: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small